Providers/Modules/Plugins/VMInsights/plugin/in_vminsights.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+
+class Fluent::VMInsights < Fluent::Input
+    require_relative 'oms_common'
+
+    require_relative 'VMInsightsDataCollector.rb'
+    require_relative 'VMInsightsEngine.rb'
+
+    Fluent::Plugin.register_input('vminsights', self)
+
+    config_param :tag, :string
+    config_param :poll, :integer, :default => 60
+    config_param :log_level, :string, :default => "info"
```

Comment:
Can you remove this from here and put it in conf as:
log_level info

Reply:
Changes made.